public/app/core/components/AppChrome/AppChrome.test.tsx

```diff
@@ -2,6 +2,7 @@ import { render, screen, waitFor } from '@testing-library/react';
 import userEvent from '@testing-library/user-event';
 import { KBarProvider } from 'kbar';
 import React, { ReactNode } from 'react';
+import { act } from 'react-test-renderer';
```

Comment:
You can import `act` from `@testing-library/react` instead.

Reply:
Ah, thanks! It's my first time using it 😅